src/rules/javascript/complex_prefs_defaults.js

```diff
@@ -0,0 +1,26 @@
+import { COMPLEX_PREFS_DEFAULTS } from 'messages/javascript';
+
+export default function(context) {
+  var filename = context.getFilename();
+
+  if (filename.indexOf('defaults/preferences/') === 0) {
+    return {
+      CallExpression: function(node) {
+        var callee = node.callee;
+        var root;
+
+        if (callee.type === 'Identifier') {
+          root = callee;
+        } else if (callee.type === 'MemberExpression') {
```

Comment:
Can neither of these be true? (`root` would be undefined then and error out below)